### ISSUE TITLE
Dropdown: Accessibility issue with the label

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#4455]
**Describe the bug**

In the dropdown component the button that triggers the custom select has an accessible name of the currently selected value (e.g. "Equals"), but does not label the purpose of the control.
Also, the label text that is announced is not unique; it should identify that it is a filter condition for the related field (e.g. "Keyword filter condition").

**Expected behavior**
In this case, the `<label>` element that accompanies the hidden `<select>` should be removed. It is not announced when a user focuses on the control, but might be announced when a user is navigating the page with the virtual cursor.
The button that triggers the dropdown must convey the purpose of the control. Text should identify which input the filter condition will apply to.

See the ARIA Authoring Practices example for a Collapsible Dropdown Listbox for an accessible implementation:

https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

**Version**
- ids-enterprise: [e.g. v4.33]

**Platform**
 - Chrome/Any

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-31955

**Comment by tmcconechy** [comment]
See also https://github.com/infor-design/enterprise/issues/4454 (point 2) which will require more testing after this change

**Comment by tmcconechy** [comment]
Lets timebox this issue in case its better to do it in the WC version

**Comment by ssaniel** [comment]
Hi Tim, This is one of the identified High Priority VPAT items. thank you!

**Comment by tmcconechy** [comment]
This issue is going to be very hard to fix (practically a partial rewrite). Maybe we can lower the priority a little as its going to take a while.

**Comment by tmcconechy** [comment]
Closing this in favor of 4844
